src/services/Store/Contract/useContracts.spec.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { renderHook } from '@testing-library/react-hooks';
+import { innerJoin, slice, prop, map, sort, pipe } from 'ramda';
+
+import { fContracts } from '@fixtures';
+import { TAddress, LSKeys, ExtendedContract, TUuid } from '@types';
+import { DataContext, IDataContext } from '../DataManager';
+import useContracts from './useContracts';
+
+const renderUseContract = ({
+  contracts = [] as ExtendedContract[],
+  createActions = jest.fn()
+} = {}) => {
+  const wrapper: React.FC = ({ children }) => (
+    <DataContext.Provider value={({ contracts, createActions } as any) as IDataContext}>
+      {' '}
+      {children}
+    </DataContext.Provider>
+  );
+  return renderHook(() => useContracts(), { wrapper });
+};
+
+describe('useContract', () => {
```

Comment:
Can you add some more unit tests for createContract , createContractWithId , and deleteContract ?